### PR TITLE
Documentation: Moved driver list to correct place and added request driver.

### PR DESCRIPTION
**Pull request by Crazometer**
### Description
Cleaned up the readme by moving the phantom driver to the correct place. Added the request driver because it should help to keep down the number of people asking about cookies and headers. Added short descriptions to help users know when a driver might help them :)

### CODE REVIEW

**Comment by Crazometer** [comment]
@Kikobeats could you take a look at this?

**Comment by Kikobeats** [comment]
Looks well, but apparently I don't have permissions for merge it.

.cc @matthewmueller

**Comment by alexgorbatchev** [comment]
@matthewmueller please merge this

File: Readme.md [hunk 1]
```diff
@@ -99,7 +99,10 @@ x(html, 'body', 'h2')(function(err, header) {
 
 ### xray.driver(driver)
 
-Specify a `driver` to make requests through.
+Specify a `driver` to make requests through. Available drivers include:
+
+- [request](https://github.com/Crazometer/request-x-ray) - A simple driver built around request. Use this to set headers, cookies or http methods.
+- [phantom](https://github.com/lapwinglabs/x-ray-phantom) - A high-level browser automation library. Use this to  render pages or when elements need to be interacted with.
```
Comment (gebrits) [inline Readme.md]: Perhaps to be extra clear: 

> Use this to  render pages or when elements need to be interacted with

To: 

> Use this to  render pages or when elements need to be interacted with, or when elements are created dynamically using javascript (e.g.: Ajax-calls)

Comment (Crazometer) [inline Readme.md]: Sounds good, I'll put that in

**Comment by matthewmueller** [comment]
looks good, thanks!

**Pull request merged**